src/utl/CMakeLists.txt

```diff
@@ -50,8 +50,8 @@ add_test(spdemo11-verify ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/cf -q test_data/sptst
 
 add_test(spdemo12 ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/spdemo -q -r 16 -right sp  test_data/sptst-r.s16 test_data/sptst-r.p16 100)
 add_test(spdemo13 ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/spdemo -q -r 16 -left  sp  test_data/sptst-l.s16 test_data/sptst-l.p16 100)
-add_test(spdemo13-verify ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/cf -q test_data/sptst-l.p16 test_data/sptst-r.p16)
-add_test(spdemo13-verify ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/cf -q test_data/sptst-l.p16 test_data/sptst-s.p16)
+add_test(spdemo13-1-verify ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/cf -q test_data/sptst-l.p16 test_data/sptst-r.p16)
+add_test(spdemo13-1-verify ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/cf -q test_data/sptst-l.p16 test_data/sptst-s.p16)
```

Comment:
hmm. I missed that one... However shouldn't the second one of the spdemo13-1-verify be spdemo13-**2**-verify? I think otherwise only the last test with the same name is executed.